test/unit/messageStoreTest.js

```diff
@@ -556,179 +558,186 @@ describe('MessageStore', () => {
-          ctx.mediatorResponse = {}
-          //Set the mediatorResponse's status
-          ctx.mediatorResponse.status = 'Successful'
-          messageStore.setFinalStatus(ctx, () => {
-
-            TransactionModel.findOne({ _id: storedTrans._id }, (err3, trans) => {
-              should.not.exist(err3);
-              (trans !== null).should.be.true()
-              trans.status.should.equal('Successful')
-              return done(err3)
-            })
-          })
-        })
-      })
-    })
-
-    it('should update the transaction status with the mediatorResponse\'s status. Case 2 -mediator status set to Failed', (done) => {
-      ctx.response = createResponse(201)
-
-      messageStore.storeTransaction(ctx, (err, storedTrans) => {
-        should.not.exist(err)
-        if (err != null) done(err)
-        ctx.transactionId = storedTrans._id
-
-        messageStore.storeResponse(ctx, (err2) => {
-          should.not.exist(err2)
-          if (err2 != null) done(err2)
-          ctx.mediatorResponse = {}
-          //Set the mediatorResponse's status
-          ctx.mediatorResponse.status = 'Failed'
-          messageStore.setFinalStatus(ctx, () => {
-
-            TransactionModel.findOne({ _id: storedTrans._id }, (err3, trans) => {
-              should.not.exist(err3);
-              (trans !== null).should.be.true()
-              trans.status.should.equal('Failed')
-              return done(err3)
-            })
-          })
-        })
-      })
-    })
-
-    return it('should truncate the response body for routes if they exceed storage limits', (done) => {
-      ctx.response = createResponse(201)
-      ctx.routes = []
-      ctx.routes.push(createRoute('route1', 201))
-      ctx.routes.push(createRoute('route2', 200))
-      for (let i = 0, end = 2000 * 1024, asc = end >= 0; asc ? i < end : i > end; asc ? i++ : i--) {
-        ctx.routes[1].response.body += '1234567890'
-      }
-
-      messageStore.storeTransaction(ctx, (err, storedTrans) => {
-        if (err) { return done(err) }
-        ctx.transactionId = storedTrans._id
-        messageStore.storeResponse(ctx, err2 =>
-          messageStore.storeNonPrimaryResponse(ctx, ctx.routes[0], () =>
-            messageStore.storeNonPrimaryResponse(ctx, ctx.routes[1], () =>
-              messageStore.setFinalStatus(ctx, () =>
-                TransactionModel.findOne({ _id: storedTrans._id }, (err3, trans) => {
-                  should.not.exist(err3);
-                  (trans !== null).should.be.true()
-                  const expectedLen = utils.MAX_BODIES_SIZE - ctx.body.length - ctx.response.body.length -
-                    ctx.routes[0].response.body.length
-                  trans.routes[1].response.body.length.should.be.exactly(expectedLen)
-                  return done()
-                })
-              )
-            )
-          )
-        )
-      })
-    })
+    // it('should truncate the response body if it exceeds storage limits', (done) => {
```

Comment:
@BMartinos lol

Reply:
oh damn! Will revert that

Reply:
These tests were testing the max storage limit of the transaction and seeing as we are removing that functionality, i have removed these tests as they are no longer relevant